CMakeLists.txt

```diff
@@ -136,6 +141,14 @@ if(ROBOTOLOGY_ENABLE_DYNAMICS)
   endif()  
 endif()
 
+# Teleoperation
+if(ROBOTOLOGY_ENABLE_TELEOPERATION)
+  find_or_build_package(retargeting)
+endif()
+
+
```

Comment:
Can you remove the unnecessary newlines? Thanks!

Reply:
done.